### PR TITLE
fixes for using FFMPEG on windows

### PR DESCRIPTION
**Pull request by blunted2night**
these allow builds on windows using latest ffmpeg

### CODE REVIEW

File: apps/openmw/mwsound/ffmpeg_decoder.cpp [hunk 1]
```diff
@@ -153,6 +153,10 @@ void FFmpeg_Decoder::open(const std::string &fname)
 
     try
     {
+        for(size_t j = 0;j < mFormatCtx->nb_streams;j++)
+            if(mFormatCtx->streams[j]->codec->codec_type == AVMEDIA_TYPE_AUDIO)
+                mFormatCtx->streams[j]->codec->request_sample_fmt = AV_SAMPLE_FMT_S16;
+
```
Comment (nikolaykasyanov) [inline apps/openmw/mwsound/ffmpeg_decoder.cpp:159]: I'm already settings sample format on line 167 (made this change for OS X)

Comment (blunted2night) [inline apps/openmw/mwsound/ffmpeg_decoder.cpp:159]: When did that change come in? I originally made this change a two or three
weeks ago, but didn't submit it until just now

On Sun, Feb 24, 2013 at 11:51 PM, Nikolay Kasyanov <notifications@github.com

Comment (nikolaykasyanov) [inline apps/openmw/mwsound/ffmpeg_decoder.cpp:159]: I took that line right from your branch to make sound work on OS X, Chris pointed me there.

**Comment by blunted2night** [comment]
Oops, I didn't notice that commit... your version of that bit of code is cleaner than mine. I'll submit a patch to clean it up.

**Pull request merged**